### ISSUE TITLE
False positive in `commented_code_linter()`

### ISSUE THREAD

**Issue by salim-b** [r-lib/lintr#1390]
If a single word is emphasized with asterisks in a comment line, `commented_code_linter()` considers the line to be R code:

``` r
tmp_file <- tempfile(fileext = "R")
writeLines(text = c("# blabla *emphasized* blabla"),
           con = tmp_file)

lintr::lint(filename = tmp_file,
            linters = list(lintr::commented_code_linter()))
#> /tmp/RtmpOpFAUM/file6626b78caaadcR:1:3: style: [commented_code_linter] Commented code should be removed.
#> # blabla *emphasized* blabla
#>   ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

<sup>Created on 2022-06-15 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by MichaelChirico** [comment]
that's because `parse(text = 'blabla *emphasized* blabla')` works... try `blabla _emphasized_ blabla` for now?

Related: #934. In general I think `commented_code_linter()` is too aggressive about identifying code, but coming up with a rule that's better than "does it parse" is tough.

**Comment by salim-b** [comment]
> that's because `parse(text = 'blabla *emphasized* blabla')` works... try `blabla _emphasized_ blabla` for now?

Ok, thanks for the explanation! Underscores don't cause the issue, yes.

**Comment by MichaelChirico** [comment]
I'll close this as subsumed under #934
